tests/test_load_xml.py

```diff
@@ -90,6 +90,7 @@ def handle_course2_errors(errorstore):
     assert_error(errorstore, FileDoesNotExist, 'html/html4.xml', "The <html url_name='html4'> tag points to the file html/htmlnotexist.html that does not exist")
     assert_error(errorstore, FileDoesNotExist, 'vertical/myvertical3.xml', "The <html> tag points to the file html/nonexistant.html that does not exist")
     assert_error(errorstore, DuplicateHTMLName, 'vertical/myvertical3.xml', "Two html tags refer to the same HTML file (using the 'filename' attribute): html/html7.html is referenced in vertical/myvertical3.xml and html/html7.xml")
+    assert_error(errorstore, EmptyTag,'problem/problem3.xml' , "The <problem url_name='problem3' display_name='Blank Common Problem'> tag is unexpectedly empty")
```

Comment:
@awaisdar001 we dont need a separate test case for this scenario. The course is already being validated. 
This new assertion/error is a `WARNING` which states that the problem is empty, hence confirming that the blank problem code was executed without any breakage.

Reply:
Sweet, thanks for the update.